editor/src/components/editor/editor-component.tsx

```diff
@@ -331,7 +332,9 @@ export const EditorComponentInner = betterReactMemo(
         <ModalComponent />
         <ToastRenderer />
         <CanvasCursorComponent />
-        {props.propertyControlsInfoSupported ? <PropertyControlsInfoComponent /> : null}
+        {props.propertyControlsInfoSupported && props.vscodeBridgeReady ? (
+          <PropertyControlsInfoComponent />
```

Comment:
Can you add a comment to this line please, because I can definitely see us forgetting why we're reliant on `props.vscodeBridgeReady` here